api/v1beta2/awscluster_webhook.go

```diff
@@ -298,5 +298,49 @@ func (r *AWSCluster) validateControlPlaneLBs() field.ErrorList {
 		}
 	}
 
+	if r.Spec.ControlPlaneLoadBalancer.LoadBalancerType == LoadBalancerTypeDisabled {
```

Comment:
Are we validating somewhere that the `type` cannot be changed once created? How are we dealing right now if a user creates the type = `disabled` and then wants `alb` and vice-versa?